### DISCUSSION TITLE
Is there a plan to develop a stand-alone IDE for Rust? Similar with goland, webstorm...

### DISCUSSION THREAD

**Discussion by CrayfishGo** [intellij-rust/intellij-rust#7942] [Q&A]
Is there a plan to develop a stand-alone IDE for Rust? Similar with goland, webstorm.

**Comment by Undin** [comment]
@CrayfishGo Could you provide details of what you expect from stand-alone Rust IDE compared with current solutions (for example, CLion + Rust plugin), please?

**Comment by LoganDark** [comment]
Your examples aren't standalone IDEs. They're forks of an IDE *system* created by JetBrains known as IntelliJ - I'm sure you've heard of it (IntelliJ IDEA for example). They're mostly language-agnostic except for the plugins that come preinstalled by default and some other architectural changes (hence they are "forks" and not just "configurations").

**Comment by kLiHz** [comment]
I think JetBrains has just released [RustRover](https://www.jetbrains.com/rust/). It's currently in preview. Also check the blog [Introducing RustRover – A Standalone Rust IDE by JetBrains](https://blog.jetbrains.com/rust/2023/09/13/introducing-rustrover-a-standalone-rust-ide-by-jetbrains/).

**Reply by LoganDark** [reply]
Can confirm, and RustRover has extra features like syntax highlighting inside `macro_rules!`. Pretty nice.
